src/screens/SSISummaryScreen/index.tsx

```diff
@@ -65,8 +66,8 @@ class SSIOnboardingSummaryScreen extends PureComponent<IProps> {
     ];
 
     return (
-      <Container>
-        <SSITabView routes={routes} />
+      <Container style={{ backgroundColor: backgrounds.secondaryDark }}>
```

Comment:
same comment here, what is the reason for adding this? and can the routes not decide which color they want? same for the styling below this